homeassistant/components/roborock/device.py

```diff
@@ -72,3 +72,13 @@ def _device_status(self) -> Status:
             if status:
                 return status
         return Status({})
+
+    async def send(
+        self,
+        command: RoborockCommand,
+        params: dict[str, Any] | list[Any] | None = None,
+    ) -> dict:
+        """Overloads normal send command but refreshes coordinator."""
```

Comment:
This is not overloading (same method but different signature), this is overriding.